src/python/fuzzing/strategy.py

```diff
@@ -87,6 +65,23 @@
     CORPUS_SUBSET_STRATEGY,
 ]
 
+# Lists of prefix and boolean strategies maintained for libFuzzer stats.
+libfuzzer_strategies_with_prefix_value = [
```

Comment:
can you capitalize these two as well? this way all the names will be consistent inside this file

Reply:
+1